src/woo_publications/api/constants.py

```diff
@@ -0,0 +1,7 @@
+from django.db.models import TextChoices
+from django.utils.translation import gettext_lazy as _
+
+
+class InformatieCategorieChoices(TextChoices):
+    waardelijst = "waardelijst", _("Waardelijst")
+    zelf_toegevoegd = "zelf_toegevoegd", _("Zelf toegevoegd")
```

Comment:
I know it's a bit of an extra hurdle, but please define translatable string literals in English, and we'll manage the translations with Django's makemessages/compilemessages.

```suggestion
    waardelijst = "waardelijst", _("Value list")
    zelf_toegevoegd = "zelf_toegevoegd", _("Custom entry")
```